src/sentry/search/events/fields.py

```diff
@@ -1314,6 +1314,24 @@ def with_default(default, argument):
     return argument
 
 
+class SnQLNumericColumnNoLookup(NumericColumn):
+    measurement_aliases = {
+        MEASUREMENTS_FRAMES_SLOW_RATE,
+        MEASUREMENTS_FRAMES_FROZEN_RATE,
+        MEASUREMENTS_STALL_PERCENTAGE,
+    }
+
+    def __init__(self, name):
```

Comment:
Since the init isn't doing anything specific to this class (ie, it's just calling `super`) you can remove this method as well

Reply:
removed!